tracer/test/Datadog.Trace.ClrProfiler.Managed.Tests/Datadog.Trace.ClrProfiler.Managed.Tests.csproj

```
@@ -15,6 +15,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.7.101.88" />
     <PackageReference Include="Confluent.Kafka" Version="1.4.3" />
     <PackageReference Include="log4net" Version="$(Log4NetVersion)" />
```

I tried to build this on .NET Framework and the new usage of Newtonsoft.Json required a package reference that was missing for the .NET Framework build (for some reason)

```suggestion
    <PackageReference Include="log4net" Version="$(Log4NetVersion)" />
    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
```